src/app/cli/interpreter.cpp

```diff
@@ -279,6 +281,7 @@ const std::map<std::string, std::string> &Interpreter::mUsageMap = *new std::map
               "panid conflict <panid>"},
     {"energy", "energy scan <channel-mask> <count> <period> <scan-duration> <dst-addr>\n"
                "energy report [<dst-addr>]"},
+    {"diag", "diag query [extmac | rloc16] <dest mesh local address>"},
```

Comment:
OpenThread uses `extaddr` for Extended Mac Address, probably better to be consistent here.
```suggestion
    {"diag", "diag query [extaddr | rloc16] <dest mesh local address>"},
```

Reply:
done, thanks